src/Microsoft.DotNet.Darc/Darc/Operations/SubscriptionOperationBase.cs

```diff
@@ -82,6 +81,16 @@ private static SubscriptionYamlData FindConflictingForwardFlowSubscription(IRead
                 && sub.TargetDirectory == updatedOrNewSubscription.TargetDirectory
                 && sub.Id != updatedOrNewSubscription.Id);
 
-    protected static string GetConfigurationFilePath(string repoUri) => SubscriptionConfigurationFolderPath / $"{GitRepoUrlUtils.GetRepoNameAndOwner(repoUri).RepoName}.yml";
+    protected static string GetConfigurationFilePath(string repoUri)
+    {
+        try
+        {
+            return SubscriptionConfigurationFolderPath / $"{GitRepoUrlUtils.GetRepoNameAndOwner(repoUri).RepoName}.yml";
+        }
+        catch (Exception)
+        {
+            return SubscriptionConfigurationFolderPath / "unknown.yml";
```

Comment:
This should not be `unknown` but maybe rather getting the last URL chunk and using that